Makefile

```diff
@@ -14,6 +14,7 @@ PYPI_VERSION     := $(shell scripts/get-py-version)
 VERSION_FLAGS    := -ldflags "-X github.com/pulumi/pulumi-terraform/pkg/version.Version=${VERSION}"
 
 build::
+	GO111MODULE=on go mod vendor
```

Comment:
We have a separate `make ensure` task that will take care of this step.  We should document that in the README/CONTRIBUTING, but for consistency with the rest of the Pulumi repos, I don't think we want to change what `make build` means for this repo.